CompuCell3D/core/CompuCell3D/steppables/PDESolvers/ReactionDiffusionSolverFVM.cpp

```diff
@@ -54,8 +55,10 @@ ReactionDiffusionSolverFVM::~ReactionDiffusionSolverFVM()
 	delete lockPtr;
 	lockPtr = 0;
 
-	delete fvMaxStableTimeSteps;
-	fvMaxStableTimeSteps = 0;
+	if(fvMaxStableTimeSteps) {
```

Comment:
I think we should be able to write a code without explicit memory allocation or use shared_ptr https://en.cppreference.com/w/cpp/memory/shared_ptr

the only downside of shared pointers is run time performance but I doubt this will be an issue here. we can fix it in the next release I am just making a comment here.